src/screens/OtherProfile/index.js

```diff
@@ -535,12 +519,25 @@ const OtherProfile = () => {
       );
     };
 
+    const onCreateChat = () => {
+      const channelName = [username, profile?.myProfile?.username].join(',');
+
+      const selectedUser = {
+        user: {
+          name: channelName,
+          image: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
+        }
+      };
+      const members = [other_id, profile.myProfile.user_id];
+      createSignChat(members, selectedUser);
+    };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a new function `onCreateChat` which creates a chat using `createSignChat`. Make sure that the `createSignChat` function handles all the parameters correctly and that it doesn't break existing functionality.
```diff
- <GlobalButton onPress={onCreateChannel}>
+ <GlobalButton onPress={onCreateChat}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->